index.html

```diff
@@ -232,39 +235,393 @@ <h3>Create</h3>
 href="#format"></a>. The creation of a DID Document is also performed by taking
 the public key value and expanding it into DID Document. An example is given
 below that expands the ed25519 <tt>did:key</tt>
-<code>did:key:z6MkpTHR8VNsBxYAAWHut2Geadd9jSwuBV8xRoAnwWsdvktH</code> into its
+<code>did:key:z6MkicdicToW5HbxPP7zZV1H7RHvXgRMhoujWAF2n5WQkdd2</code> into its
 associated DID Document:
         </p>
 
         <pre class="example" title="A DID Document derived from a did:key">
           {
             "@context": [
               "https://www.w3.org/ns/did/v1",
-              "https://w3id.org/security/suites/ed25519-2018/v1",
-              "https://w3id.org/security/suites/x25519-2019/v1"
+              "https://w3id.org/security/suites/ed25519-2020/v1",
```

Comment:
Ah, yes, good catch. 

I had it on my list to specify how the `@context` is generated, but forgot to do that in my bleary-eyed state last night. I'll add how `@context` is generated in the next pass to this PR, most likely using the suggestions you mentioned on how to generate `@context`.

What I'm trying to go for w/ `Multikey` and (ideally, `JsonWebKey`) is to remove all of the per-cryptosuite JSON-LD Contexts... so, a did:key DID Document would only need ONE `@context`... "https://www.w3.org/ns/did/v2" (note the "v2" at the end). That context would define `Multikey` and `JsonWebKey` and that's most likely all we'd need for public key extensibility... no more date-versioned cryptosuite contexts.

However, I can't make that change yet because we don't have buy-in from the VC2WG or DIDWG yet... so, this iteration of the algorithm is a bit more funky than I'd like. It's not ideal, I'm just trying to capture what's out there today with what the ideal design is that we're trying to move towards.

Reply:
We can defer with an issue, like you did elsewhere for JWK.

Reply:
I went ahead and defined enough to make the example in the spec valid now: 2d2e1016b2aa90094cd68a6c30fc166c2b0f8de1

I couldn't make Multikey work because it's not defined in any context yet, but noted that we expect it to be defined by VC2WG in the next year or so.